### ISSUE TITLE
Custom Sensor not loading properly

### ISSUE THREAD

**Issue by joshwapohlmann** [lgsvl/simulator#670]
I tried implementing a custom sensor, building upon the [ComfortSensor](https://github.com/lgsvl/ComfortSensor) using the [information in the docs](https://www.lgsvlsimulator.com/docs/sensor-plugins/).
After successfully compiling the sensor, I get a NullReferenceException.
Turned out one item in the Sensor list stored in `Simulator.Web.Config.Sensors` is null.
This lead to `RuntimeSettings.Instance.SensorPrefabs` having the same issue.

**Comment by ntutangyun** [comment]
could you provide more details like simulator logs ?

**Comment by joshwapohlmann** [comment]
After creating this issue, I found that I failed to update the Sensor Prefab with the Sensor Component attached.

So for people coming after me or the dev team implementing some safety measures:
Your sensor WILL compile as an asset bundle even if you failed to add the Sensor Component to your prefab gameobject. If you happen to find a `null` value in your sensors list it is because while loading the prefab no Component matching `SensorBase` could be found. 

**Currently no failsafe will inform you about this.**
